### PR TITLE
CustomerOrder bugged with loadFile 

### PR DESCRIPTION
**Pull request by ssok305**
LoadFile is unable to read CSV correct, as tested before I'm getting back 0 from the size().

### CODE REVIEW

File: CustomerOrder [hunk 1]
```diff
@@ -0,0 +1,140 @@
+/**
+ * The finance, marketing, and sales departments want to understand customers better but need the historical data to do so.
+ * They want to have a database/pseudo-DB that stores all of the customer order
+ * information along with the time and date of their orders.
+ * [This database will grow each simulated “day”.]
```
Comment (daedren30) [inline CustomerOrder:5]:
```suggestion
 * This database will grow each simulated day.
```

**Review by daedren30 — DISMISSED** [review]
Looks good. Made a few changes from "OrderItems" to "OrderItem" and different code to parse the CSV. Also removed brackets and quotes on "day" in comments because it was throwing some kind of character error. It should work now.

**Review by daedren30 — APPROVED** [review]
Looks good. Made a few changes to fix loadfile, removed "s" from OrderItems, and removed brackets/quote from a comment.

**Review by jblss888 — APPROVED** [review]
there are too many changes to the original pull for me to try to copy and run in on my computer too verify that the load works. However, from what I read in the code and from freeman's comment it should work. (if not I will it once the code is up) the skeleton looks good and shows the changes we all discussed in our meeting>

**Pull request merged**